src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/BasePostsCreator.java

```diff
@@ -70,9 +73,14 @@ public PostsCreator closeGuidelinesMessage() {
 
   @Override
   public CategoryPills clickAddCategoryButton() {
-    getAddCategoryButton().click();
-    wait.forElementVisible(By.cssSelector("." + getBaseCssClassName() + " .pop-over-compass"));
-
+    try {
+      getAddCategoryButton().click();
+      wait.forElementVisible(By.cssSelector("." + getBaseCssClassName() + " .pop-over-compass"));
+      categoryPills.setEmpty(false);
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)